include/rocksdb/utilities/cache_dump_load.h

```diff
@@ -67,6 +67,10 @@ class CacheDumpReader {
 // dump or load process related control variables can be added here.
 struct CacheDumpOptions {
   SystemClock* clock;
+  // Max duration time in ms for dumper or loader
+  uint64_t max_duration_time_ms = 0;
```

Comment:
How about `std::chrono::microseconds deadline = std::chrono::microseconds::zero();` (copied this from elsewhere in the public API)? It is less prone to overflow and unit mistakes.